libraries/rush-sdk/src/index.ts

```diff
@@ -84,3 +126,34 @@ for (const property in rushLibModule) {
     });
   }
 }
+
+function requireRushLibUnderFolderPath(folderPath: string): RushLibModuleType {
+  const rushLibModulePath: string = Import.resolveModule({
+    modulePath: RUSH_LIB_NAME,
+    baseFolderPath: folderPath
+  });
+
+  return require(rushLibModulePath);
+}
+
+function findRushJsonLocation(startingFolder: string): string | undefined {
```

Comment:
```suggestion
/**
 * @privateRemarks
 * Keep this in sync with `RushConfiguration.tryFindRushJsonLocation`.
 */
function findRushJsonLocation(startingFolder: string): string | undefined {
```

Can you add a similar note in the doc comment of `RushConfiguration.tryFindRushJsonLocation`?

Reply:
Sure